srcpkgs/android-file-transfer-linux/template

```diff
@@ -1,16 +1,30 @@
 # Template file for 'android-file-transfer-linux'
 pkgname=android-file-transfer-linux
-version=4.2
-revision=4
+version=4.3
+revision=1
 build_style=cmake
 configure_args="-DBUILD_SHARED_LIB=1"
 hostmakedepends="qt5-qmake qt5-host-tools ninja pkg-config"
 makedepends="file-devel fuse-devel qt5-devel readline-devel
  qt5-tools-devel"
-depends="qt5-svg"
+depends="qt5-svg $pkgname-cli $pkgname-libs"
```

Comment:
slight preference for not doing $pkgname replacement here

Reply:
Done, thanks.